src/dashboard/osdeps/Ubuntu-14.json

```diff
@@ -0,0 +1,13 @@
+{
+  "_comments": [
+    "Add any comments here",
+    "Include here keys, repos, and packages required by this component not already in archivematicaCommon osdeps file"
+  ],
+  "osdeps_repokeys": [
+  ],
+  "osdeps_repos": [
+  ],
+  "osdeps_packages": [
+  { "name": "gettext", "state": "latest"}
```

Comment:
I think some of the dependencies listed in the current control files may not be required. I tested deploying archivematica from source with the dependencies in the osdeps files in this PR and so far haven't found issues.